# Notes

## Exploring bacterial diversity via a curated and searchable snapshot of archived DNA sequences

### Blackwell, 2021

-   Characterize bacterial genomes with uniform standardized approach.

-   Ease of finding things out about genomes, searching.

-   Good for finding similarities between genomes, plasmids/SNPs, current databases/resources can't do that well

-   Uniform, searchable assemblies, technical barrier lowered.

-   COBS index allows for searching for SNPs, specific genes, plasmids, break into k-mers

-   MinHash searches for similar genomes.

-   pp-sketch index calculates genetic distances between assemblies.

-   90.6% of the data set came from 20 species, (data set from ENA sequence archive, public genomic data sequenced by others)

-   Used to find antimicrobial resistance genes in bacteria, how many.

-   Find multi-class resistant genomes/species (resist 3+ classes of antimicrobials)

-   Some biases present in data, bias towards medically relevant bacteria, MCR genomes, can't do prevalence estimates

-   Want to sample and sequence many different bacteria to improve data, remove biases, get from diverse environments and global locations to accurately study gene flow.
